index.html

```diff
@@ -269,9 +269,9 @@ <h3>Subscriptions</h3>
                     <div class="feature-description">
                         <p><strong>Never miss updates again:</strong></p>
                         <ul>
-                            <li>Subscribe a channel with one click</li>
-                            <li>Use your custom feed</li>
-                            <li>Watch your favourite artists</li>
+                            <li>Subscribe to a channel with one click</li>
+                            <li>Use your customized feed</li>
```

Comment:
Custom feeds are feeds users created themselves. Customized feeds are feeds created by third parties with changes from the users.

The question is: are they *custom* or *customized*?

Reply:
Well, that is customized, since feed is the the whats new area, right?

I didnt realized I switched the entries around, but thats not a problem, right?

Reply:
The feed will receive a major update soon: https://github.com/TeamNewPipe/NewPipe/pull/2309
I'd suggest to alter this slide once that PR is merged.

Reply:
That looks great.

Whatever you want, not my call ;P next update sounds good